src/Util.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace SilverStripe\Assets;
+
+class Util
```

Comment:
The methods on this class are only used by `FlysystemAssetStore`, so you may as well just move this methods to that class. And then delete this class

You can update the existing `FlystemAssetStore::isSeekableStream()` to the logic currently in this class

Reply:
We use static method `rewindStream` in `Sha1FileHashingService` and `FlysystemAssetStore`.
And `isSeekableStream` in `FlysystemAssetStore` and `Util` uses this method itself.